source/_static/scss/includes/_theme.scss

```diff
@@ -111,6 +111,9 @@ $theme-properties: (
     --alert-warning-code-bg: #f3dfe1 #3d1014,
     --alert-warning-link-decoration-color: #e3c3c6 #54141a,
 
+    // Recommended videos
+    --recommended-videos-marker: url("../img/icons/play.svg") url("../img/icons/play-dark.svg"),
```

Comment:
I like the concept of the icons for each video.
But this particular icon looks pretty pixelated on my screen.

Can we use a different icon image set?

Reply:
@djwfyi Can you share a screenshot? Its an SVG, so not its sure why it's getting pixelated.

Reply:
Here's a screenshot.
![image](https://user-images.githubusercontent.com/53910321/218514641-3850a9a1-5631-465a-b5e6-535d9f851d2a.png)

Reply:
It may just be my monitor. It's better on the MacBook's screen than on the external.

Reply:
Thanks @djwfyi, I will try to reproduce the issue.